mne/beamformer/_lcmv.py

```diff
@@ -131,6 +131,8 @@ def make_lcmv(info, forward, data_cov, reg=0.05, noise_cov=None, label=None,
 
     data_cov = pick_channels_cov(data_cov, include=ch_names)
     Cm = data_cov['data']
+    if 'estimator' in data_cov:
+        del data_cov['estimator']
```

Comment:
Why this?

Reply:
Because the estimator can contain `sklearn` classes, and isn't preserved on `write_cov / read_cov` round-trip anyway.